pkg/cgu/cgu.go

```diff
@@ -26,6 +34,63 @@ type CguBuilder struct {
 	errorMsg string
 }
 
+// NewCguBuilder creates a new instance of CguBuilder.
+func NewCguBuilder(
+	apiClient *clients.Settings,
+	name, nsname string,
```

Comment:
Are all these arguments are required in order to create CGU object? NewBuilder function accepts only minimum required parameters for the object. All optional parameters have to be implemented using With* functions

Reply:
Not all the parameters are required. I'll update and create With* functions for the optional ones.

Reply:
@kononovn Done, I updated the NewCguBuilder to include only required arguments and added With functions for optional parameters.

Reply:
All arguments fit one line. Please refactor.

Reply:
Updated.